source/common/runtime/runtime_features.cc

```diff
@@ -131,5 +135,25 @@ RuntimeFeatures::RuntimeFeatures() {
   }
 }
 
+bool RuntimeFeatures::enabledByDefault(absl::string_view feature) const {
```

Comment:
Hmmm, I totally want to make you document this, and we don't have QUIC docs yet.
Can you TODO(12923) to document this magic prefix somewhere where in Envoy runtime?